ample/util/mrbump_util.py

```diff
@@ -186,18 +198,27 @@ def _extractPurged(self, mrbump_dir):
                 purged_results[d['ensemble_name']] = d
         return purged_results
 
-    def extractResults(self, mrbump_dir, purge=False):
+    def extractResults(self, mrbump_dir, purge=False, debugPrint=False):
         if not mrbump_dir or not os.path.isdir(mrbump_dir):
             raise RuntimeError("Cannot find mrbump_dir: {0}".format(mrbump_dir))
         purged_results = {}
         if purge:
             purged_results = self._extractPurged(mrbump_dir)
+        originalLogLevel = None
+        if not debugPrint:
```

Comment:
I think you might want to do something like this instead
https://stackoverflow.com/a/11029841/3046533

Reply:
Not sure I understand - isn't that what I'm doing? The only difference is that I'm resetting the level after the call? It's a bit convoluted but I didn't want to turn off debug print for the whole module as it's useful, but need to turn it off when checking runs so it doesn't create huge logfiles.

Reply:
you are but it looks wrong. I linked to the post because you may be able to disable logging for parts/functions of the module. alternatively, I would refactor this out and create a logging context, such that you could do 
```python
with logging_suppressed(debug_print):
    self._extractResults(mrbump_dir, archived_ensembles=purged_results.keys())
```

Reply:
e.g., https://gist.github.com/simon-weber/7853144

Reply:
You're right - a perfect use case for a context manager! Addressed with e5658c1